dvc/system.py

```diff
@@ -10,7 +10,11 @@
 
 logger = logging.getLogger(__name__)
 
-if platform.system() == "Windows" and sys.version_info < (3, 8):
+if (
+    platform.system() == "Windows"
+    and sys.version_info < (3, 8)
+    and sys.getwindowsversion() > (6, 2)
```

Comment:
Should this be `>= (6, 2)`?
I see that [Windows8 is 6.2](https://en.wikipedia.org/wiki/Windows_NT#Releases).

Reply:
Yes you're right, it should (although I haven't tested it on Windows 8)